compositor_pipeline/src/pipeline/graphics_context.rs

```diff
@@ -8,7 +8,9 @@ pub struct GraphicsContext {
     pub queue: Arc<wgpu::Queue>,
 
     #[cfg(feature = "vk-video")]
-    pub vulkan_ctx: Option<Arc<vk_video::VulkanCtx>>,
+    pub vulkan_device: Option<Arc<vk_video::VulkanDevice>>,
```

Comment:
maybe group that together into a single option?
- `Option<(Device, Instance)>` 
- `Option<VulkanCtx>`